templates/summarize-anthropic/summarize_anthropic.ipynb

```diff
@@ -127,6 +127,11 @@
     " \n",
     "We can also use remote runnable to call it.\n",
     "\n",
+    "In `server.py`, set -\n",
```

Comment:
not sure this is the best place to put this, but i think general some of these notebooks need cleaning up